src/main/resources/com/google/cloud/spanner/jdbc/DatabaseMetaData_GetColumns.sql

```diff
@@ -27,6 +27,8 @@ SELECT TABLE_CATALOG AS TABLE_CAT, TABLE_SCHEMA AS TABLE_SCHEM, TABLE_NAME, COLU
     WHEN SPANNER_TYPE LIKE 'STRING%' THEN -9
     WHEN SPANNER_TYPE = 'JSON' THEN -9
     WHEN SPANNER_TYPE = 'TIMESTAMP' THEN 93
+    WHEN SPANNER_TYPE = 'UUID' THEN 1111
```

Comment:
Type code `1111` is the JDBC-code for `Other`. There is no specific type code for UUID in JDBC.